troposphere/static/js/components/admin/ImageRequest.react.js

```diff
@@ -136,141 +136,143 @@ export default React.createClass({
 
         return (
         <div className="request-admin pull-right admin-detail">
-            <div>
-                <h3>Request Information</h3>
+            <div style={{ marginBottom: "20px" }}>
+                <h3 className="t-body-2">Request Information</h3>
                 <div>
-                    Request ID:
-                    {request.get("id")}
+                   Request ID:
+                   {` ${request.get("id")}` }
```

Comment:
From a language analysis point of view. A common way to analyze a feature is to look at what it can do that you /couldn't/ do before. Because I can take any expression with backticks and convert it into one with regular quotes and some plus symbols, I know it doesn't increase the expressiveness of the language. It really is a readability thing. If that's the case and all you have to gain is readability then my earlier comment is valid.